sykepengeperioder/src/main/kotlin/no/nav/helse/sparkel/sykepengeperioder/infotrygd/Utbetalingshistorikk.kt

```diff
@@ -25,19 +25,21 @@ class Utbetalingshistorikk(jsonNode: JsonNode, sjekkStatslønn: Boolean = false)
         .map { Inntektsopplysninger(it) }
         .filter(Inntektsopplysninger::skalTilSpleis)
 
+    private val arbeidsKategoriKode = jsonNode["arbeidsKategoriKode"].asText()
+
     private val ferie1: Utbetaling? = jsonNode["ferie1Fom"]?.takeUnless { it.isNull }?.textValue()?.let(LocalDate::parse)?.let { ferie1fom ->
         jsonNode["ferie1Tom"]?.takeUnless { it.isNull }?.textValue()?.let(LocalDate::parse)?.let { ferie1tom ->
-            Utbetaling(ferie1fom, ferie1tom, "", "", ferie1fom, 0.0, "9", "Ferie", "0")
+            Utbetaling(ferie1fom, ferie1tom, "", "", ferie1fom, 0.0, "9", "Ferie", "0", arbeidsKategoriKode)
         }
     }
 
     private val ferie2: Utbetaling? = jsonNode["ferie2Fom"]?.takeUnless { it.isNull }?.textValue()?.let(LocalDate::parse)?.let { ferie2fom ->
         jsonNode["ferie2Tom"]?.takeUnless { it.isNull }?.textValue()?.let(LocalDate::parse)?.let { ferie2tom ->
-            Utbetaling(ferie2fom, ferie2tom, "", "", ferie2fom, 0.0, "9", "Ferie", "0")
+            Utbetaling(ferie2fom, ferie2tom, "", "", ferie2fom, 0.0, "9", "Ferie", "0", arbeidsKategoriKode)
         }
     }
 
-    val utbetalteSykeperioder = jsonNode["utbetalingList"].map { Utbetaling(it) } + listOfNotNull(ferie1, ferie2)
+    val utbetalteSykeperioder = jsonNode["utbetalingList"].map { Utbetaling(it, arbeidsKategoriKode) } + listOfNotNull(ferie1, ferie2)
```

Comment:
Jeg har mappa inn koden i per utbetaling, men jeg ser at vi her har et objekt på utsiden som holder på litt mer innhold. Kanskje jeg skal legge det der?

Reply:
Flytta opp `arbeidsKategoriKode` til hvert element av "Sykepengehistorikk" istedet for ned i perioden. Det ser bedre ut!